download_traffic_data.py

```
@@ -0,0 +1,234 @@
+                print(f"  ... and {len(archives) - 12} more archives")
+            return
+        
+        # Download measurements
+        if not args.location_id:
+            print("\n🚗 JARTIC Traffic Data Download")
+            print("="*60)
+            print(f"Period: {args.start.strftime('%Y-%m-%d')} to {args.end.strftime('%Y-%m-%d')}")
+            
+            # Calculate which archives we need
+            months_needed = []
+            current = datetime(args.start.year, args.start.month, 1)
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)